pylintrc

```diff
@@ -102,6 +102,8 @@ disable=
     consider-using-enumerate,  # pylint 2.1, clean up tests later
     no-else-raise, # python 2.4.0
     import-outside-toplevel, # pylint 2.4.2
+    f-string-without-interpolation,  # pylint 2.5.0, bare f-strings are ok
```

Comment:
Yes, bare f-strings are ok in general, but they could hide bugs.

For example,
```python
subst = "WORLD"
s = f"HELLO,subst"
```
What if I just forgot to add braces? Reviewers (people) don't see that, I think.
There is only one error of this type in code.

Reply:
What about 
```
subst = "WORLD"
s = "Hello {subst}"
```
Is the second line a bug or not?

IPA is going to use more f-strings over time. CPython upstream is considering to make all strings f-strings and introduce a new prefix for plain strings. We (CPython core) are encouraging people to use f-strings everywhere.

Reply:
Of course, two examples can be buggy: missing `f` or `{}`, and neither default type of string helps.
The difference is that `pylint` can prevent the bug in my example, but can't in yours.

Reply:
@tiran, I like 'f-strings'.
It seems that my argument is too weak for CPython core :)